### PR TITLE
Neo-Russkiya

### PR DESCRIPTION
**Pull request by Agameofscones**
Adds ~~three~~ ~~two~~ one languages in respect to the two big non NT players on the galactic stage, the ~~[Elysium Republic](https://nanotrasen.se/wiki/index.php/Lore)~~, and the [Union of Soviet Socialist Planets](https://nanotrasen.se/wiki/index.php/Union_of_Soviet_Socialist_Planets).

But why you may ask? "_...going so far as to appropriate the USSR’s flag, anthem, and language._"
Lore accuracy, promotion of greater roleplay options.
I mean seriously, who would genuinely know clownish over a real language from a real political entity right?

These are character generation options.

~~Elysiun uses `?`~~
Neo-Russkiya uses `>`
these were the best fit keys that weren't already mapped.

Neo-Russkiya: _"Neo-Russkiya, a bastard mix of Gutter, Sol Common, and old Russian. The official language of the USSP. It has started to see use outside of the fringe in hobby circles and protest groups. The linguistic spirit of Sol-Gov criticisms."_
image

:cl: Triiodine
add: Neo-Russkiya Language
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Hory shet these can have some ridiculously long words. That's WAY too long on some of these, wadda fuk

What is `ivanoolamyetskeybotatyevtglozdoomatskybotatodaskymochbarslovglozodasky`?

I dig the meme factor but fuck if someone walks down the hall saying one-word screen fillers I'm not even going to bother looking at chat anymore

**Comment by Agameofscones** [comment]
Aye, I'll update the spacing for the Russian variants, in review the length is indeed quite silly.

**Comment by KasparoVy** [comment]
Thanks!

**Comment by Agameofscones** [comment]
Feedback is the spice of life 😄

**Comment by taukausanake** [comment]
This is kind of interesting. I'm not against it but it's kind of weird seeing it. I'm all for having Buzzsky speak a different language though!

**Comment by Agameofscones** [comment]
Hey lads, would y'all be more in support of this if I **removed** Traditional Russkiya? In review having an event only language that _requires_ VV isn't exactly a brilliant call 😅

File: code/modules/mob/language.dm [hunk 1]
```diff
@@ -426,6 +426,50 @@
...
+	name = "Traditional Russkiya"
+	desc = "A more formalized form of Russkiya that attempts to (poorly) adhere to old Russian grammar and rules. The official langauge of the USSP."
+	colour = "trad_srus"
+	key = ">"
+	flags = RESTRICTED
+
+/datum/language/elysian
+	name = "Elysian"
+	desc = "A secretive winding language rarely spoken outside the Elysium Republic. It's often considered rude to speak in this tongue due to the Elysium Republic's stance on non-humans."
+	speech_verb = "dulciquents"
```
Comment (Citinited) [inline code/modules/mob/language.dm]: My googling seems to imply that this (along with `interpollates` and `sussurates`) is spelt wrongly. Are you sure these are correct?

Comment (Agameofscones) [reply code/modules/mob/language.dm]: I can't believe I missed that, my bad! Thanks for pointing that out.

**Comment by Agameofscones** [comment]
Love you too Travis. I :heart: map failures without map changes.

**Comment by Shadeykins** [comment]
Not sure on Elysiun since it deliberately exists in lore as a fringe entity (teeny), but Neo-Russkiya is pretty cool.

**Comment by KasparoVy** [comment]
Davaj davajjj 🇷🇺🚀🛸🇷🇺

**Comment by variableundefined** [comment]
I concur with Shadey that Elysian shouldn't be included because they're an obscure entity in lore.

What purposes does adding more languages to the game serve besides more russian joke?

**Comment by Agameofscones** [comment]
> I concur with Shadey that Elysian shouldn't be included because they're an obscure entity in lore.
> 
> What purposes does adding more languages to the game serve besides more russian joke?

In review, I think I'd definitely agree that Elysiun wouldn't make sense given their isolation and small reference in the very well hidden lore timeline on the wiki. I'd be perfectly comfortable slimming down this PR and axing it.

In regards to space Russian, I'd argue; There have been USSP events run by staff, there is a Russian station as a every round space ruin, there are Russian outfits as equipment presets for events and Abus, they've their own dedicated wiki page and appear on the lore timeline a few times.

And in-character, at least to me, it'd make much more sense to see a heated debate on the bridge between a bunch of people speaking Russian than hearing the filtered zubs and norfs of a deeply political Clownish discussion.

The idea of Space Russia and the USSP is very heavily ingrained in the game's content and context. I think reinforcing it with language would be a neat way to wrap up the package.

**Comment by variableundefined** [comment]
I'm good with leaving Russkiya in but leaving Elysian out.

**Comment by Agameofscones** [comment]
I'll clean up the PR later today, gotta get them there midterm studying hours in. 👍

**Comment by KasparoVy** [comment]
Variety ftw.

**Comment by Ozewse** [comment]
I'm down. +1

**Comment by KasparoVy** [comment]
I'm `taganidivechnetzvech`. :ok_hand:

**Comment by KasparoVy** [comment]
DAVAJ DAVAAAAJ

**Comment by kwawakari** [comment]
This merge makes me feel very ginoolamvichmyaksokollinkanetz

**Pull request merged**